higlass/server.py

```diff
@@ -205,6 +210,23 @@ class ServerError(Exception):
     pass
 
 
+def get_free_socket(sock_dir):
+    for p in count():
```

Comment:
why not create a unique socket name using something like `slugid.nice()`?

Reply:
Actually I didn't think of that 😅  Plus this method has the advantage of automatically cleaning up dead sockets from the past. If you still prefer `slugid` I'll use that.

Reply:
It's the same to me. Just was curious :-)